src/config.ts

```diff
@@ -1,3 +1,9 @@
+// FIXME: in the Swarm Extension process is not defined and would throw an error
+// see https://github.com/ethersphere/bee-dashboard/issues/518
+export function getEnvVariable(variableName: string): string | undefined {
+  if (typeof process === 'object') return process.env[variableName]
```

Comment:
Did you test this for the normal Dashboard? I remember that the test `typeof process` did not work with the CRA mechanism of the env. replacement. Maybe you have to extend it to `process.env`...

Reply:
You are right. However, the test `process && process.env` complaints in the Swarm Extension so I wonder if removing the env variables here and injecting them instead would not be easier.